source/developers/references/developer_guide/internationalization/i18n.rst

```diff
@@ -407,9 +407,9 @@ everything (scraping and serving) is working properly.
 
 First, use the coverage tool to generate dummy files.
 
-.. code-block:: sh
+.. code-block:: bash
```

Comment:
nit: Is there a reason to change this from `sh` to `bash` ?

Reply:
Just wanted to make it unified across the documentation, since there's no `.. code-block:: sh` and it's `bash` everywhere.